acceptance/openstack/sharedfilesystems/v2/sharenetworks_test.go

```diff
@@ -0,0 +1,23 @@
+package v2
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+)
+
+func TestShareNetworkCreateDestroy(t *testing.T) {
```

Comment:
ah it's because the previous branch has been squashed and merged as one commit, so gets a new hash